compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -16549,12 +16553,22 @@ OMR::Z::TreeEvaluator::inlineVectorBinaryOp(TR::Node * node, TR::CodeGenerator *
          mask4 = getVectorElementSizeMask(node);
          breakInst = generateVRRcInstruction(cg, op, node, targetReg, sourceReg1, sourceReg2, 0, 0, mask4);
          break;
-      // These require mask4 = 0x3, other values illegal
-      // These only operate on long BFP elements,
+      /*
+       * Before zNext, these required mask4 = 0x3 and other values were illegal
+       * This was because they only operated on long BFP elements.
+       * From zNext onward, mask4 is variable and sets the element size to operate on
+       */
```

Comment:
This comment was to update a preexisting comment. I can also add a similar comment to "getSupportsOpCodeForAutoSIMD". Should I also remove the comment from this location?